### DISCUSSION TITLE
Bash or Zsh?

### DISCUSSION THREAD

**Discussion by pakhomovvv** [Cyfrin/foundry-full-course-cu#2480] [Q&A]
ChatGPT tells me that **zsh** is better or my mac OS 11 Big Sur, but pat is using bash, so idk what should i use

**Comment by skushagra9** [comment]
use zsh with ohmyzsh plugin

**Reply by pakhomovvv** [reply]
i cannot find a plugin called ohmyzsh, i found zsh debug, but idk if this is useful

**Comment by EngrPips** [comment]
I couldn't find it too @skushagra9
